puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testRuby.groovy

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent { label 'fast' }
```

Comment:
I think slow nodes will do ;)

Reply:
In the past we've seen the job timeout and fail so I wasn't sure.

Reply:
indeed, but that isn't related to compute power, but (AFAICT) some timeout when facter wants to connect to some Amazon API when resolving a fact.

Reply:
It could be IO related, but I'll remove it for now since we need to get the job working again anyway.